api/envoy/config/trace/v3/trace.proto

```diff
@@ -23,14 +23,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Tracing]
 // Tracing :ref:`architecture overview <arch_overview_tracing>`.
 
-// The tracing configuration specifies global
-// settings for the HTTP tracer used by Envoy. The configuration is defined by
-// the :ref:`Bootstrap <envoy_api_msg_config.bootstrap.v3.Bootstrap>` :ref:`tracing
-// <envoy_api_field_config.bootstrap.v3.Bootstrap.tracing>` field. Envoy may support other tracers
-// in the future, but right now the HTTP tracer is the only one supported.
+// The tracing configuration specifies settings for an HTTP tracer provider used by Envoy.
+//
+// Envoy may support other tracers in the future, but right now the HTTP tracer is the only one
+// supported.
 message Tracing {
```

Comment:
Is this message actually used anymore or just the inner HTTP message? Should this message actually be tagged for deletion with the inner message moved somehow? I don't think the tool supports this but we might want to file an issue here? cc @htuch

Reply:
`config.trace.v3.Tracing` will not be used anymore in `v4`.

I don't see a support for marking the entire message type as `deprecated`. The closest example is `v2.Bootstrap.Runtime` which only has `(deprecated)` comment in the docs.

I've added an "attention" message to `config.trace.v3.Tracing` for now.

@htuch Should I open an issue or you can phrase it better what needs to be done ?

Reply:
Please open an issue to track this. We will need some tooling work I think to make sure this message gets deleted/moved/etc. I'm not sure the best way of handling.

Reply:
opened #10740